catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -414,7 +414,7 @@
     </feature>
 
     <feature name="catalog-app" install="auto" version="${project.version}"
-             description="The DDF Catalog provides a framework for storing, searching, processing, and transforming information.\nClients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.\nAt the core of the Catalog functionality is the Catalog Framework, which routes all requests and responses through the system, invoking additional processing per the system configuration.::DDF Catalog">
+             description="The DDF Catalog provides a framework for storing, searching, processing, and transforming information.\nClients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.\nAt the core of the Catalog functionality is the Catalog Framework, which routes all requests and responses through the system, invoking additional processing per the system configuration.::Catalog">
```

Comment:
This would probably be good to do project wide. We have "DDF" in metatype titles and descriptions as well.